modules/tinymce/src/core/main/ts/caret/CaretContainer.ts

```diff
@@ -49,9 +49,9 @@ const insertInline = (node: Node, before: boolean): Node => {
     }
 
     if (node.nextSibling) {
-      parentNode.insertBefore(textNode, node.nextSibling);
+      parentNode?.insertBefore(textNode, node.nextSibling);
```

Comment:
Not convinced this is really the right solution here... but not sure what else we should do other than nothing in this case.

Reply:
I feel like not blowing up may hide problems and cause other side effects, but it may also lead to a better user experience 🤷‍♂️